components/developer/dejagnu/Makefile

```diff
@@ -53,4 +54,5 @@ ASLR_MODE = $(ASLR_ENABLE)
 REQUIRED_PACKAGES += shell/expect
```

Comment:
Please remove this line.  It is duplicate.

Reply:
done